WooCommerce/Classes/POS/Presentation/Barcode Scanning/BarcodeScannerContainer.swift

```diff
@@ -103,7 +104,7 @@ class BarcodeScannerHostingController: UIHostingController<EmptyView> {
         /// or change between the `began` call and the `ended` call.
         /// It's better practice for barcode scanning to only consider the presses when they end.
         for press in presses {
-            guard let key = press.key?.charactersIgnoringModifiers else { continue }
+            guard let key = press.key else { continue }
```

Comment:
We pass the raw key now, to allow us to test the modifier/special key handling.

Note that the previous use of `charactersIgnoringModifiers` meant we used the raw keystroke, e.g. `2` instead of `@`, which could break some edge case workflows with 2D codes.